### ISSUE TITLE
[Scheduling Tool] Mandatory Description

### ISSUE THREAD

**Issue by ronniegeraghty** [Azure/azure-sdk-tools#7120]
I believe we should make the Description field mandatory when scheduling a meeting with the scheduling tool. I often see meetings from partner teams with no description and no materials which makes it very difficult to determine the context for the meeting.

**Comment by maririos** [comment]
@JonathanCrd because the Description comes from a library component, is it correct to say that we either do Description required for all or we won't be able to do this?

**Comment by JonathanCrd** [comment]
> @JonathanCrd because the Description comes from a library component, is it correct to say that we either do Description required for all or we won't be able to do this?

No, we can require it for specific meeting types, since the check logic is in the submit button of each app.

**Comment by JonathanCrd** [comment]
> But what about UI? can we add the text Required or the * to some but not all?
I honestly think it should be ok to make it required for all, but was wondering....

You have a good point. Adding the required `*` mark is not a substantial change anyway, we can just simply add it on top of the component or make it a property of the component itself.

To be honest, I don't think we will need this component at all in the future once we start working on the full UI/UX redesign. I'm thinking of better approaches now that I have more experience with the platform.
